.pre-commit-config.yaml

```diff
@@ -15,6 +15,10 @@ repos:
     rev: 2.0.0
     hooks:
       - id: bashate
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.8.3
+    hooks:
+      - id: flake8
   - repo: local
```

Comment:
Could you please add the [autopep8 hook](https://github.com/pre-commit/mirrors-autopep8)

```YAML 
 - repo: https://github.com/pre-commit/mirrors-autopep8
    rev: v1.5.4
    hooks:
      - id: autopep8
```